<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hyperfile</class>
 <widget class="QWidget" name="Hyperfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="IPFS">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Hyperfile IPFS File Upload</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
          <widget class="QLabel" name="hyperfileicon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../bitcoin.qrc">:/icons/hyperfile</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Innova's &lt;span style=&quot; font-weight:600;&quot;&gt;Hyperfile&lt;/span&gt; is our&lt;span style=&quot; font-weight:600;&quot;&gt; decentralized file uploading service&lt;/span&gt; using IPFS. IPFS is the Interplanetary Decentralized File System, Hyperfile utilizes the IPFS APIs, Public Gateways, and Infura's API to upload and pin files into space free and forever. You can utilize Hyperfile POD to upload files and also timestamp them with the Innova blockchain.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Use this on the decentralized web at: &lt;a href=&quot;https://ipfs.infura.io/ipfs/QmdfxKo6HVgnegahHCXn4X3HoMg7NR3LwCTjCAyxgmCZic&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#ffea00;&quot;&gt;Infury&lt;/span&gt;&lt;/a&gt; | &lt;a href=&quot;https://github.com/innova-foundation/infury&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#fbff00;&quot;&gt;Infury Github&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;More Information on IPFS: &lt;a href=&quot;https://ipfs.io&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#fff700;&quot;&gt;https://ipfs.io&lt;/span&gt;&lt;/a&gt; | &lt;a href=&quot;https://infura.io&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#f3ff00;&quot;&gt;https://infura.io&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
      </layout>
   </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Upload Form</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="labelFile">
              <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Select a file location</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="filePushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Select File</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="createPushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>350</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>UPLOAD TO IPFS</string>
                </property>
                <property name="icon">
                 <iconset resource="../bitcoin.qrc">
                  <normaloff>:/icons/hyperfile</normaloff>:/icons/hyperfile</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="createPodButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>350</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>UPLOAD TO IPFS AND PAY 0.001 INN TO POD/TIMESTAMP</string>
                </property>
                <property name="icon">
                 <iconset resource="../bitcoin.qrc">
                  <normaloff>:/icons/hyperfile</normaloff>:/icons/hyperfile</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="hashLabel">
           <property name="text">
            <string>IPFS CID/Hash:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="hashLabel_2">
           <property name="text">
            <string>Innova Hyperfile POD Address:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="hashLabel_3">
           <property name="text">
            <string>Innova Hyperfile POD Transaction ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="View">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="checkButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>255</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>View Uploaded File on Infura</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="checkButtonCloudflare">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>255</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>View Uploaded File on Cloudflare</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="checkHashButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>View Hashed INN Timestamp on Explorer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     </layout>
    </widget>
   </item>
  </layout>
  <widget class="QLabel" name="labelAlerts">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16</width>
     <height>19</height>
    </rect>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, stop:0 #F0D0A0, stop:1 #F8D488); color:#000000;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
